ui/eureka/eureka.js

```diff
@@ -477,7 +477,7 @@ let Options = {
           trackerName: {
             en: 'Strider',
             de: 'Läufer',
-            fr: 'Trotteur',
+            fr: 'Simurgh',
```

Comment:
This is incorrect, I think.  The tracker name here is the name from https://ffxiv-eureka.com/.  Start a tracker, click on the green clipboard icon, and you'll get something like `NMs dont la fenêtre est disponible : Pampa → Seigneur → Teles → Empereur → Callisto → Number → Jaha → Amemet → Caym → Bomba → Serket → Julika → Cavalier → Poly → Trotteur → Hazmat → Fafnir → Amarok → Lamashtu → Pazuzu (Vents violents dans 37m)`

In this case, Trotteur is the name.